airflow/www/static/js/utils.js

```diff
@@ -41,3 +41,23 @@ export const finalStatesMap = () => new Map([
   ['skipped', 0],
   ['no_status', 0],
 ]);
+
+// Get hidden and visibility strings depending on browser
+export const getVisibilityVars = () => {
+  let hidden;
+  let visibilityChange;
+  if (typeof document.hidden !== 'undefined') { // Opera 12.10 and Firefox 18 and later support
```

Comment:
What browsers do we actually need this conditional for? Opera 12 and FF 18 are Oooold.